src/cmd/usb-util.js

```diff
@@ -1,5 +1,6 @@
 const { getDevice, isDeviceId } = require('./device-util');
 const { systemSupportsUdev, promptAndInstallUdevRules } = require('./udev');
+const ui = require('../lib/ui');
```

Comment:
This `ui` is a class so has to be an instance in order to use the methods
Something like this:

```
const UI = require('../lib/ui');
const ui = new UI();
```

It would be good if we inject ui in the functions. In that way we can get the use from `cmd` classes.

Reply:
A second thought over here is: we can make that `src/lib/ui` returns as default something like `new UI()` in that way there gonna be just one instance in case we need it in different files on the same execution flow hehe.